xdp-dump/tests/test-xdpdump.sh

```diff
@@ -93,6 +95,8 @@ test_interfaces()
 
 test_capt_pcap()
 {
+    assert_kernel_version_greater_or_equal 5.7.0
```

Comment:
This test should pass, if it does not something is wrong.

Reply:
This one can handle legacy programs, but only when using newer kernels (together with the ```-p``` option). Using 5.6 it fails with ```ERROR: Can't load eBPF object: Kernel verifier blocks program loading(-4007)```. Same goes for test_capt_term, test_exitentry, test_snap, test_promiscuous and test_pname_parse. Looking into the verbose output, the problem seems to be using a helper function ```bpf_xdp_output``` missing in the older kernels
 ```
        ; bpf_xdp_output(xdp, &xdpdump_perf_map,
        32: (18) r2 = 0xffff8c1cdd423000
        34: (b7) r5 = 20
        35: (85) call unknown#121
        invalid func unknown#121
```

Reply:
If this is the case , i.e., bpf_xdp_output() missing, then xdp-dump will not work at all, and all tests can be skipped. However as @tohojo mentioned we should not use plain version numbers to check this.